src/main/java/service/DataParserImpl.java

```diff
@@ -0,0 +1,33 @@
+package service;
+
+import java.util.ArrayList;
+import java.util.List;
+import model.FruitDataDto;
+
+public class DataParserImpl implements DataParser {
+
+    public static final String COMMA = ",";
+    public static final int OPERATION_INDEX = 0;
+    public static final int FRUIT_INDEX = 1;
+    public static final int QUANTITY_INDEX = 2;
+    public static final int DATA_INDEX = 1;
+    public static final String WHITE_SPACE = " ";
+    public static final int FIRST_LINE_INDEX = 0;
```

Comment:
if you don't use these constants outside of this class make them private, if you use them don't redeclare same variable several times